cmd/kubeapps-apis/_test/mock_plugin.go

```diff
@@ -0,0 +1,273 @@
+/*
+Copyright © 2021 VMware
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package _test
```

Comment:
Sorry, my reply had a typo... I had meant write `server_test` (ie. rather than having it as part of the `server` package directly as earlier), but it probably belongs best as `plugin_test`? Anyway, the key point is that the module ends in `_test`, but it shouldn't just be `_test`.

Reply:
Ah, got it! I assumed it was a sort of convention for mocks :D, no problem, moving to `plugin_test` instead.